### PR TITLE
Add backend support for maximum allowed runtime

### PR DESCRIPTION
**Pull request by kdryja**
This is to address https://github.com/StefanRudvin/CodeMarkerBack/issues/6 
+CC @jhudsy 

Two things are introduced here:

#### First is the fix for the time always being zero:
This was mostly caused by base docker image from ubuntu not having `bs` utility, so technically this commit doesn't fully solve the issue. Although as I was investigating, I also discovered some extra issues with the Decimal field, since there was a discrepancy between `max_digits` and `decimal_places` - the former has to be greater than the latter.

I'll remotely install bs utility on the docker image, after this pull request is merged

#### Add support for maximum runtime calculation:

The tests will now also be checked against maximum runtime property. If the runtime exceeds the pre-defined value by the course coordinator, 0 marks is going to be awarded. I'll open separate PR for changes in the frontend.

Btw I've accidentally run my formatter, thus lot of noise-changes (i.e. indentation lvl), plz ignore, I was too lazy to undo it.

### CODE REVIEW

**Comment by kdryja** [comment]
Also sneaked in fix for https://github.com/StefanRudvin/CodeMarkerBack/issues/5 
Seems that Django's automatic PUT resets attributes back to default, if they are not present in the PUTed form, so instead I just manually coded the method to handle put request (instead of relying on django's implementation)

File: app/models.py [hunk 1]
```diff
@@ -27,57 +28,57 @@ class Course(models.Model):
...
 class Resource(models.Model):
-    filename = models.FileField(upload_to='resources/')
+    filename = models.FileField(upload_to="resources/")
 
-    status = EnumField(choices=['start', 'in_progress', 'complete'])
-    language = EnumField(choices=['python2', 'python3', 'java', 'cpp', 'c', 'ruby'],
-                         default='python2')
+    status = EnumField(choices=["start", "in_progress", "complete"])
+    language = EnumField(
+        choices=["python2", "python3", "java", "cpp", "c", "ruby"], default="python2"
```
Comment (StefanRudvin) [inline app/models.py:52]: We should refactor these into a separate enum

**Review by StefanRudvin — APPROVED** [review]
This code takes me back! Good times 👍

File: app/models.py (hunk 1 above)
Comment (StefanRudvin) [reply app/models.py:52]: If we ever want to refactor haha

**Pull request merged**